azure-pipelines.yml

```diff
@@ -0,0 +1,91 @@
+# Go
+# Build your Go project.
+# Add steps that test, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/go
+
+trigger:
+- master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+variables:
+  GOBIN:  '$(GOPATH)/bin' # Go binaries path
+  GOROOT: '/usr/local/go' # Go installation path
+  GOPATH: '$(system.defaultWorkingDirectory)/gopath' # Go workspace path
+  modulePath: '$(GOPATH)/src/github.com/$(build.repository.name)' # Path to the module's code
+  GOOS: 'linux'
+  GO111MODULE: 'on'
+  GO_PROJ: 'github.com/Azure/$(build.repository.name)'
+
+steps:
+- script: |
```

Comment:
From https://github.com/Microsoft/azure-pipelines-image-generation/issues/741 it looks like Azure DevOps will have Go 1.12 in the ubuntu image - so we can remove this line -- it'd speed up the pipeline!  (Just something to keep an eye on)